test/no-generic-link-name-config-override.md

```diff
@@ -0,0 +1,17 @@
+# Links with descriptive and non-descriptive names
```

Comment:
In newer tests, I try to make the top-level heading text match the file name, but in its readable form.